lib/buffer-search.coffee

```diff
@@ -58,6 +59,17 @@ class BufferSearch
       ResultsMarkerLayersByEditor.set(editor, layer)
     layer
 
+  checkSanity: (findPattern) ->
```

Comment:
I picked the name to allow for additional checks to be added to it as needed, but if you think that's unlikely I'm happy to rename.